README.md

```diff
@@ -10,51 +10,88 @@ __The project is in its *very* early stages, there is nothing to see here yet.__
 
 Tobira consists of two parts:
 
+- `frontend` contains the web interface written in TypeScript/React.
 - `backend` contains the video portal backend written in Rust which takes care of data persistence and communication to Opencast.
-- `frontend` contains the web interface written in Typescript/React.
 
 
 ## How to build Tobira
 
-### Install Development Dependencies
+### 1. Development Tools
 
-You will need `rustc` and `cargo` to build the Tobira backend.
-To get them, use [`rustup`](https://rustup.rs) or your package manager.
```

Comment:
Why remove the hint towards `rustup`? :thinking:

Reply:
Why don't we explain how to get npm?
It really depends on your system how to best get this.
The important information here is that this is a rust tool and you need cargo for building the backend.
That should be enough information for developers to get started.
We should avoid having unnecessary or distracting information in here.

I made an exception for floof since that's a somewhat obscure tool no one knows about